### PR TITLE
Feature/co 169 error

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by brandonlee503** [comment]
Your pull req looks good to me!

**Comment by ghost** [comment]
thanks for taking a look at this @brandonlee503

File: src/main/groovy/edu/oregonstate/mist/sadapi/resources/SadResource.groovy [hunk 1]
```diff
@@ -27,70 +29,113 @@ class SadResource {
...
-                         @QueryParam('termCodeEntry') String termCodeEntry,
-                         @QueryParam('applNo') Long applNo,
-                         @QueryParam('seqNo') Long seqNo) {
-        Object result = sadDAO.delete(pidm, termCodeEntry, applNo, seqNo)
-        if (!result) {
-            throw new WebApplicationException(Response.Status.NOT_FOUND)
+    public Response delete(@PathParam('pidm') Long pidm,
+                  @QueryParam('termCodeEntry') String termCodeEntry,
+                  @QueryParam('applNo') Long applNo,
+                  @QueryParam('seqNo') Long seqNo) {
```
Comment (ghost) [inline src/main/groovy/edu/oregonstate/mist/sadapi/resources/SadResource.groovy]: inconsistent leading whitespace

File: src/main/groovy/edu/oregonstate/mist/sadapi/resources/SadResource.groovy [hunk 2]
```diff
@@ -27,70 +29,113 @@ class SadResource {
...
+                         @QueryParam('seqNo') Long seqNo) {
+        ResponseBuilder responseBuilder
         if (termCodeEntry && applNo && seqNo) {
-            result = sadDAO.queryOne(pidm, termCodeEntry, applNo, seqNo)
-            if (!result) {
-                throw new WebApplicationException(Response.Status.NOT_FOUND) }
+            Sad result = sadDAO.queryOne(pidm, termCodeEntry, applNo, seqNo)
+            if (result) {
+                responseBuilder = Response.ok()
+                responseBuilder.entity(result)
```
Comment (ghost) [inline src/main/groovy/edu/oregonstate/mist/sadapi/resources/SadResource.groovy]: `responseBuilder = ok(result)`

**Comment by ghost** [comment]
note that internal errors generated by dropwizard do not follow our json guidelines. additional minor development would be necessary to figure out how to format general error messages.

note: skeleton?

**Pull request merged**